src/error.ts

```diff
@@ -6,6 +6,7 @@
 
 import type {Failure} from './event.js';
 import {JsonFile} from './util/package-json-reader.js';
+import * as pathLib from 'path';
```

Comment:
Minor nit, I happened to choose the `pathlib` casing in a bunch of other modules.